### ISSUE TITLE
Ajax Modal not closable & Icons not appearing

### ISSUE THREAD

**Issue by sys-arch** [DNA-styx/SBPP-Windmill-Dashboard#62]
The modals that notify you of AJAX errors block the site and do not allow you to close them so a button or icon had to be added to close them

The transparent button/link on the top right, called Menu, does not work or does nothing, whether you are logged in or not.

![imagen](https://github.com/user-attachments/assets/5695efae-fe68-46d7-908c-e74338789a57)

**Comment by DNA-styx** [comment]
Thanks for reporting (and the screenshot!)

AJAX: Will have a look. Do you know if there is an easy way to force that error to appear?

Icons: There's a function breaking upstream change with SourceBan 1.8.x that has broken the icons (#61) . Have a fix in dev, will push a new beta out with a fix shortly.

**Comment by sys-arch** [comment]
Yes, that thing you show on screenshot is what not working. Maybe you can change it to put login or admin menu (or both, show login when is not logged in and admin menu when it is)

**Comment by DNA-styx** [comment]
Thanks for confirming @sys-arch .

I've tweaked how the links work, they are now called from a separate file. 

![image](https://github.com/user-attachments/assets/6dbc6168-c4a2-4054-9d3f-c12e3a9a50f2)

I've also added a go in/out to the drop down as you suggested. I've also added an admin panel link to see if it is useful. Tried the entire admin menu, but it seemed a bit too much.

![image](https://github.com/user-attachments/assets/f1bd4351-88f7-442a-8d95-276f2e204736)

![image](https://github.com/user-attachments/assets/882abf30-606a-4291-891f-77dad0388623)

**Comment by sys-arch** [comment]
Since this is abandoned, i will closed it as "Not fixed"
